### ISSUE TITLE
Document need to install libffi-dev (and libssl-dev)

### ISSUE THREAD

**Issue by giovannipizzi** [aiidateam/aiida_core#746]
It looks like new versions of paramiko depend on cffi, that in turn requires libffi-dev (and maybe also libssl-dev).
If this is the case, we should document this (E. Bosoni reported this by installing with  `pip install -e aiida_core[atomic_tools,docs]`)

**Comment by giovannipizzi** [comment]
Confirmed, also to avoid insecurity warnings

**Comment by ltalirz** [comment]
@asle85 since the anaconda error popped up on your machine, could you please try whether giovanni's suggestion fixes it?
I will now have a look at the issue concerning libffi-dev

**Comment by ltalirz** [comment]
starting from fresh Ubuntu-16.04 (ubuntu-desktop installed), following installation instructions
```
sudo apt-get install git python2.7-dev python-pip virtualenv postgresql postgresql-server-dev-all postgresql-client
git clone https://github.com/aiidateam/aiida_core
cd aiida_core
git fetch origin release_v0.10.0
git checkout release_v0.10.0
sudo pip install -e .[atomic_tools,docs]
...
Successfully installed Jinja2-2.9.5 Mako-1.0.7 MarkupSafe-0.23 PyCifRW-3.6.2.1 PyMySQL-0.7.9 Pygments-2.2.0 SQLAlchemy-1.0.12 SQLAlchemy-Utils-0.31.2 Sphinx-1.5.2 aiida-core alabaster-0.7.10 aldjemy-0.6.0 alembic-0.9.6 amqp-1.4.9 anyjson-0.3.3 ase-3.12.0 asn1crypto-0.23.0 babel-2.5.1 backports.functools-lru-cache-1.4 backports.shutil-get-terminal-size-1.0.0 billiard-3.3.0.23 celery-3.1.25 certifi-2017.7.27.1 cffi-1.11.2 chardet-3.0.4 click-6.7 cryptography-2.1.1 cycler-0.10.0 decorator-4.1.2 django-1.7.11 django-extensions-1.5.0 docutils-0.13.1 ecdsa-0.13 enum34-1.1.6 ete3-3.0.0b35 frozendict-1.2 future-0.16.0 futures-3.1.1 idna-2.6 imagesize-0.7.1 ipaddress-1.0.18 ipython-5.5.0 ipython-genutils-0.2.0 kombu-3.0.37 matplotlib-2.1.0 meld3-1.0.0 monty-1.0.1 numpy-1.12.0 palettable-3.1.0 paramiko-2.1.2 passlib-1.7.1 pathlib2-2.3.0 pexpect-4.2.1 pickleshare-0.7.4 plumpy-0.7.10 portalocker-1.1.0 prompt-toolkit-1.0.15 psycopg2-2.7.1 ptyprocess-0.5.2 pyasn1-0.3.7 pycparser-2.18 pycrypto-2.6.1 pydispatcher-2.0.5 pymatgen-4.5.3 pyparsing-2.2.0 python-dateutil-2.6.0 python-editor-1.0.3 python-mimeparse-0.1.4 pytz-2014.10 pyyaml-3.12 reentry-1.0.2 requests-2.18.4 scandir-1.6 scipy-0.19.1 seekpath-1.6.0 setuptools-36.6.0 simplegeneric-0.8.1 singledispatch-3.4.0.3 six-1.10.0 snowballstemmer-1.2.1 spglib-1.9.10.1 sphinx-rtd-theme-0.1.9 subprocess32-3.2.7 supervisor-3.1.3 tabulate-0.7.5 traitlets-4.3.2 tzlocal-1.3 ujson-1.35 uritools-1.0.2 urllib3-1.22 validate-email-1.3 voluptuous-0.8.11 wcwidth-0.1.7
```
So I guess Ubuntu-16.04 is not affected (?)
How do I reproduce the problem and what should the instructions for the user be?

**Comment by asle85** [comment]
On my laptop (OS Ubuntu 16.04) python was already installed through Anaconda, and I got the error  `version 'GFORTRAN_1.4' not found`.  With Giovanni's suggestion I was able to install AiiDA.

**Comment by ltalirz** [comment]
Ok, it seems giovanni forgot to add the `ssh_kerberos` extra. This leads to an error `    Package gss was not found in the pkg-config search path`
The error is fixed by `sudo apt-get install libffi-dev libkrb5-dev` as written in the `setup_requirements.py`. @sphuber is going to add this to the installation instructions.

**Comment by ltalirz** [comment]
@asle85 thanks!
@sphuber For the conda thing, perhaps just add one entry to the troubleshooting list. Something like:

* When installing AiiDA via the [conda package manager](https://conda.io/), you may encounter an error similar to `version 'GFORTRAN_1.4' not found (required by /usr/lib/libblas.so.3)`.
This can be fixed by `export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libgfortran.so.3` as suggested on the corresponding [conda issue](https://github.com/ContinuumIO/anaconda-issues/issues/686)

**Comment by sphuber** [comment]
Fixed in PR #832
